{"uid":"56d83eb753740ff3","name":"test_database_relation","fullName":"tests.integration.test_charm#test_database_relation","historyId":"a46dd5fec5170ae1b5112e2bb246aae9","time":{"start":1751334303229,"stop":1751335322642,"duration":1019413},"description":"Test the database relation.","descriptionHtml":"<p>Test the database relation.</p>\n","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_charm.py\", line 96, in test_database_relation\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 3209, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nTimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1751334303011,"stop":1751334303011,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1751334303011,"stop":1751334303011,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1751334303012,"stop":1751334303228,"duration":216},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1751334303228,"stop":1751334303228,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1751334303229,"stop":1751334303229,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test the database relation.","status":"broken","statusMessage":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_charm.py\", line 96, in test_database_relation\n    await ops_test.model.wait_for_idle(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 3209, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nTimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","steps":[],"attachments":[{"uid":"896b9d16f8b0c0ba","name":"log","source":"896b9d16f8b0c0ba.txt","type":"text/plain","size":8153}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"ops_test::finalizer","time":{"start":1751335322675,"stop":1751335323351,"duration":676},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1751335322675,"stop":1751335322675,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1751335323351,"stop":1751335323351,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1751335323351,"stop":1751335323351,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1751335323351,"stop":1751335323352,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_charm"},{"name":"host","value":"pkrvmbietmlfzoi"},{"name":"thread","value":"19335-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"ab3e09c7c201183e","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":104,"skipped":0,"passed":241,"unknown":30,"total":375},"items":[{"uid":"68116d395bbc7240","reportUrl":"../2886//#testresult/68116d395bbc7240","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: mysql-k8s/1","time":{"start":1751247762242,"stop":1751247882275,"duration":120033}},{"uid":"5c1c66f2ff0014b8","reportUrl":"../2885//#testresult/5c1c66f2ff0014b8","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: mysql-k8s/1","time":{"start":1751161387571,"stop":1751161502977,"duration":115406}},{"uid":"3e56b375a39ad3e9","reportUrl":"../2884//#testresult/3e56b375a39ad3e9","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1751074938419,"stop":1751075952150,"duration":1013731}},{"uid":"a169b50f20a14a75","reportUrl":"../2882//#testresult/a169b50f20a14a75","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: mysql-k8s/2","time":{"start":1750988516996,"stop":1750988629485,"duration":112489}},{"uid":"698c2305a1047a8b","reportUrl":"../2880//#testresult/698c2305a1047a8b","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: mysql-k8s/0","time":{"start":1750902150410,"stop":1750902261854,"duration":111444}},{"uid":"7d75a0741a78ecb","reportUrl":"../2879//#testresult/7d75a0741a78ecb","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1750815840047,"stop":1750816853099,"duration":1013052}},{"uid":"4e5cb7d8bb3d1791","reportUrl":"../2876//#testresult/4e5cb7d8bb3d1791","status":"passed","time":{"start":1750729439028,"stop":1750729702190,"duration":263162}},{"uid":"e8ccce89e05636fa","reportUrl":"../2874//#testresult/e8ccce89e05636fa","status":"passed","time":{"start":1750642864765,"stop":1750643137007,"duration":272242}},{"uid":"6e081f6e12b51dd9","reportUrl":"../2873//#testresult/6e081f6e12b51dd9","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [idle] active:","time":{"start":1750556633006,"stop":1750557645473,"duration":1012467}},{"uid":"34964c052f71a003","reportUrl":"../2872//#testresult/34964c052f71a003","status":"passed","time":{"start":1750470210886,"stop":1750470880267,"duration":669381}},{"uid":"8b8f86eb7a612c94","reportUrl":"../2871//#testresult/8b8f86eb7a612c94","status":"passed","time":{"start":1750383789098,"stop":1750384061767,"duration":272669}},{"uid":"7bcc6db15c138503","reportUrl":"../2870//#testresult/7bcc6db15c138503","status":"passed","time":{"start":1750297547808,"stop":1750297818998,"duration":271190}},{"uid":"c29fe806168a3ceb","reportUrl":"../2869//#testresult/c29fe806168a3ceb","status":"passed","time":{"start":1750210984794,"stop":1750211535774,"duration":550980}},{"uid":"c227779537a8bdc6","reportUrl":"../2865//#testresult/c227779537a8bdc6","status":"passed","time":{"start":1750124598094,"stop":1750125038069,"duration":439975}},{"uid":"c689185a1c69c559","reportUrl":"../2864//#testresult/c689185a1c69c559","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1750038093780,"stop":1750039103374,"duration":1009594}},{"uid":"903f051955c6ab5a","reportUrl":"../2863//#testresult/903f051955c6ab5a","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1749951914744,"stop":1749952933387,"duration":1018643}},{"uid":"918817b097d763d8","reportUrl":"../2862//#testresult/918817b097d763d8","status":"passed","time":{"start":1749865357257,"stop":1749865893124,"duration":535867}},{"uid":"89738995cd4a9b6d","reportUrl":"../2859//#testresult/89738995cd4a9b6d","status":"passed","time":{"start":1749779125780,"stop":1749779399800,"duration":274020}},{"uid":"5168f682c341c79f","reportUrl":"../2856//#testresult/5168f682c341c79f","status":"broken","statusDetails":"TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1749692731959,"stop":1749693742228,"duration":1010269}},{"uid":"bd54c1652e7cc8b1","reportUrl":"../2854//#testresult/bd54c1652e7cc8b1","status":"passed","time":{"start":1749606279380,"stop":1749606558003,"duration":278623}}]},"tags":["asyncio","abort_on_fail"]},"source":"56d83eb753740ff3.json","parameterValues":[]}